src/templates/layouts/pressRelease/index.test.tsx

```diff
@@ -248,8 +235,30 @@ describe('<pressRelease> with valid data', () => {
   window.print = jest.fn()
   test('renders the print button and handles click', () => {
     render(<PressRelease {...data} />)
-    const printButton = screen.getByText('Print')
+    const printButton = screen.getByTestId('print-button')
     fireEvent.click(printButton)
     expect(window.print).toHaveBeenCalled()
   })
+  test('does not render Media contacts section when the contact is archived', () => {
+    // This is the state of data at the component when the author is archived
+    const archivedAuthorContacts = [
+      {
+        id: '96668498-d442-4b55-8b14-7ec90410f418',
+        name: null,
+        description: null,
+        numbers: [
+          {
+            id: null,
+            type: null,
+            number: null,
+            ext: null,
+          },
+        ],
+        email: null,
+      },
+    ]
```

Comment:
So when the author is archived, all the fields are set to `null`? 🤔 Is that just a Drupal thing, or..? How do you know?

Reply:
I am not sure where this is happening to tell you the truth. I don't see where it is happening in next-build at a glance but I also can't imagine Drupal doing it like this. I'll dig around.